src/preset-themes/005.json

```diff
@@ -0,0 +1,43 @@
+{
+  "colors": {
+    "toolbar": {
+      "h": 33,
+      "s": 72,
+      "l": 92,
+      "a": 100
+    },
+    "toolbar_text": {
+      "h": 38,
+      "s": 97,
+      "l": 66,
+      "a": 100
+    },
+    "accentcolor": {
+      "h": 127,
+      "s": 24,
+      "l": 90,
+      "a": 100
+    },
+    "textcolor": {
+      "h": 181,
+      "s": 42,
+      "l": 60,
+      "a": 100
+    },
+    "toolbar_field": {
+      "h": 32,
+      "s": 0,
+      "l": 99,
+      "a": 36
+    },
+    "toolbar_field_text": {
+      "h": 38,
+      "s": 96,
+      "l": 65,
+      "a": 100
+    }
+  },
+  "images": {
+    "headerURL": "./bg-16.svg"
+  }
+}
```

Comment:
Also, not sure if it's strictly "required" or not, but this file is missing a `title` attribute (says my 🗑:fire: schema linter).